### PR TITLE
Replace fetch.dart with new http library and roll mojo

### PR DESCRIPTION
**Pull request by collinjackson**

### CODE REVIEW

File: sky/packages/sky/lib/src/services/image_cache.dart [hunk 1]
```diff
@@ -5,9 +5,10 @@
 import 'dart:async';
 import 'dart:ui' as ui;
 
+import 'package:flutter/http.dart';
+
```
Comment (abarth) [inline sky/packages/sky/lib/src/services/image_cache.dart]: No blank line here.

**Comment by abarth** [comment]
LGTM

**Comment by Hixie** [comment]
Travis is unhappy.

**Comment by abarth** [comment]
LGTM

**Pull request closed without merging**